Frameworks/Foundation/NSUserDefaults.mm

```diff
@@ -58,6 +59,9 @@ - (instancetype)init {
 
     _registrationDict = [NSMutableDictionary dictionary];
 
+    _synchronizeQueue = [NSOperationQueue new];
```

Comment:
Looks like I need .attach

---
In reply to: [69994676](https://github.com/Microsoft/WinObjC/pull/612#discussion_r69994676) [](ancestors = 69994676,69786087)